style.css

```diff
@@ -21,12 +21,52 @@
 
 body {
   max-width: 450px;
-  overflow: hidden;
+  overflow-x: hidden;
```

Comment:
- [ ] kindly consider adding this overflow-x on your common styling 
of your page to avoid the scroll on x axis
 hint you can do so by 
```
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  font-family: 'Inter', serif;
overflow-x:hidden;
}
```
the after that consider reducing the width of your header to make your page have no x scroll totally